widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -226,6 +134,7 @@ public static void onRequestPermissionsResult(int requestCode, String[] permissi
      *
      * <p>This method is no-op for other non-Glia triggered results.</p>
      */
+    //TODO check the possibilities to remove this method
```

Comment:
The same as Alex asked for another TODO, is this going to be done in this PR or should we create a separate Jira ticket for this?

Reply:
I will remove TODOs, I think it makes sense to have separate tickets for these two functions.
Do you think it makes sense to include this in this project?
This should be unused since we introduced `ActivityWatchers` for screen sharing and permissions, but unfortunately, we forgot to deprecate it.

Reply:
I think a separate ticket is better. It doesn't seem to be essential for our current project.

Reply:
I will mark these methods as deprecated since we're not using them anymore.